src/lib.rs

```diff
@@ -69,9 +69,9 @@ struct Wildcard {
 
 type BraceStack = ArrayVec<(u32, u32), 10>;
 
-pub fn glob_match(glob: &str, path: &str) -> bool {
-  let glob = glob.as_bytes();
-  let path = path.as_bytes();
+pub fn glob_match<T: AsRef<[u8]>>(glob: T, path: T) -> bool {
```

Comment:
This is a backwards-incompatible change. Previously, `(&String, &str)` was accepted (via implicit deref coercion from `&String` to `&str`), but now it’s rejected because using the same type parameter `T` twice forces the second argument to be `&String` as well.

Reply:
Thanks for pointing this out, I overlooked this issue.